packages/flutter/lib/src/services/binding.dart

```diff
@@ -37,6 +37,7 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
     SystemChannels.system.setMessageHandler((dynamic message) => handleSystemMessage(message as Object));
     SystemChannels.lifecycle.setMessageHandler(_handleLifecycleMessage);
     SystemChannels.platform.setMethodCallHandler(_handlePlatformMessage);
+    SystemChannels.textInput.setMethodCallHandler((MethodCall call) => Future<void>.value());
```

Comment:
This looks very odd - and hopefully isn't the real fix here. If it is, this will need a comment explaining what's going on.

Reply:
With Scribble I think one will be able to enter text when `TextInput._instance` isn't initialized, so the text input plugin now may start the communication (before that the communication starts when a text field gains focus). So I guess we can do this, or use a separate method channel for Scribble.

Reply:
Also I suspect flutter iPad Scribble does not work until the first text field gains focus, because `TextInput._instance` is lazily initialized?

Reply:
So the issue is that the indirect scribble interaction is set up on the engine side, but with nothing on the framework side handling messages on the textinput channel yet, there are a few moments where iPadOS isn't sure whether a scribble interaction is going to happen or not, so it swallows the inputs and shows a little pencil streak.

If there's a default handler on the method channel, as here, then that doesn't happen because the engine gets a very fast reply over the channel and can let iPadOS know that there won't be a scribble interaction occurring.

I suppose an alternative solution might be to wait to add the UIIndirectScribbleInteraction until we get a message from the framework over the channel...I think that might work in 2.12, since it no longer waits for focus to set up the `TextInput._instance`.

If this is acceptable, though, I can try to distill that down into something appropriate for a comment.

Reply:
> I think that might work in 2.12, since it no longer waits for focus to set up the `TextInput._instance`.

Ah I see `TextInput` is accessed in `initState`. 

I would slightly prefer setting up the method handler eagerly over waiting for the first incoming message in the engine. @goderbauer suggestions for making it less odd? Maybe we can expose a `TextInput.ensureInitialized` method and call that?

Reply:
I don't fully understand how the scribble feature works, but it is odd that the engine sends us a message and we just drop it on the floor. We should either set up the correct message handler here and handle the message or change the engine side to not send these over the channel until the framework can actually handle them.